nlp/seq2seq/util.py

```diff
@@ -0,0 +1,59 @@
+###
```

Comment:
Why are we adding this model in **util.py** file?

Reply:
`Seq2SeqFairseqWrapper` is a kind of utility. Any seq-2-seq model can be used through this. In future ... we can use something else than fairseq and just add a wrapper class as a utility in it.

Reply:
Ok.